docs/windows.md

```diff
@@ -0,0 +1,29 @@
+# Windows Installation
+
+## Overview
+
+The installer will install Grafana Agent into the default directory `C:\Program Files (x86)\Grafana Agent`. [Windows Exporter](https://github.com/grafana/windows_exporter) can be enabled with all default options. 
+
+## Installation
+
+After installation ensure that you can reach `http://localhost:12345/-/healthy` and `http://localhost:12345/agent/api/v1/targets`. 
+
+If Grafana Agent is re-installed and an agent-config.yaml already exists it will not overwrite the existing one.
+
+After installation adjust agent-config.yaml to meet specific needs and restart the agent to see changes.
+
+## Silent Installation
+
+Silent installation can be achieved via  `grafana-agent-installer.exe /S  /EnableExporter "true"`. EnableExporter enables or disables Windows Exporter, default is `false`.
+
+## Security
+
+The Agent configuration is installed alongside the Agent itself, by default. Depending on your configuration, you may not want that for security reasons and may instead want to make it protected. The configuration is by default stored in `C:\Program Files (x86)\Grafana Agent`. You can change the configuration location by running `sc config "Grafana Agent" binpath= "<installed_directory>\agent-windows-amd64.exe -config.file=\"<new_path>\agent-config.yaml\""` in cmd as an admin.
+
+## Uninstall
+
+Via Remove Programs or uninstaller.exe in the directory the Agent is installed in. This will turn off and remove the Agent then delete any installed files in the applications directory.
```

Comment:
```suggestion
If the Grafana Agent is installed using the installer, it can be uninstalled via Windows' Remove Programs or `C:\Program Files (x86)\Grafana Agent\uninstaller.exe`. Uninstalling the Agent will stop the service and remove it from disk. This will include any configuration files in the installation directory. 
```